pwa/src/templates/templateParts/filters/verticalFilters/VerticalFiltersTemplate.tsx

```diff
@@ -415,10 +415,9 @@ export const VerticalFiltersTemplate: React.FC<VerticalFiltersTemplateProps> = (
                     onChange={() => setStatusRadioFilter(status.value)}
                     key={status.value}
                   >
-                    <input
-                      type="radio"
+                    <RadioButton
                       value={status.value}
-                      {...register("status")}
+                      // {...register("status")}
```

Comment:
This comment can be deleted altogether (we don't need to register this field). Same goes for the other RadioButtons.